src/Common/src/System/Net/NetworkInformation/UnixCommandLinePing.cs

```diff
@@ -18,7 +18,7 @@ internal static class UnixCommandLinePing
 
         private static readonly string s_discoveredPing4UtilityPath = GetPingUtilityPath(ipv4: true);
         private static readonly string s_discoveredPing6UtilityPath = GetPingUtilityPath(ipv4: false);
-        private static readonly bool s_isOSX = RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
+        private static readonly bool s_isBSD = RuntimeInformation.IsOSPlatform(OSPlatform.OSX) || RuntimeInformation.IsOSPlatform(OSPlatform.Create("FREEBSD"));
```

Comment:
Why does this use OSPlatform.Create("FREEBSD") rather than OSPlatform.FreeBSD?

Reply:
tests include this file and build as netstandard. OSPlatform.FreeBSD is not available there.